text_generation/causal_lm/cpp/continuous_batching/library/src/continuous_batching_pipeline.cpp

```diff
@@ -55,18 +56,18 @@ class ContinuousBatchingPipeline::Impl {
     }
 
 public:
-    Impl(const std::string& models_path, const SchedulerConfig& scheduler_config) {
+    Impl(const std::string& models_path, const SchedulerConfig& scheduler_config, const std::string device = "CPU", const ov::AnyMap& plugin_config = {}) {
```

Comment:
I suppose in `Impl` we don't need default arguments, because they are always passed via `ContinuousBatchingPipeline` ctor